### DISCUSSION TITLE
Getting different distributions of variables

### DISCUSSION THREAD

**Discussion by sbphysics** [MadAnalysis/madanalysis5#195] [Q&A]
Dear organizers,
I am new to MadAnalysis. I am facing a problem which I cannot understand how to resolve. I have produced one scalar with two invisible particles having similar masses from p-p collision considering a point-like interaction. While plotting DeltaR distributions between two invisibles or one scalar-one invisible, I am getting similar distributions for all and the distributions instruct that every particle is back-to-back. I cannot understand how it is possible. I am getting similar issue for DeltaPhi distribution also. I have used the MadAnalysis built in Madgraph and obtained the MA files before running with Pythia. Is there any basic error that I am making? Any help is appreciated.

Thanks in advance
SB

**Comment by BFuks** [comment]
Dear SB,

It will be hard for us to help you without more details. What you observed may or may not be what you expected, depending on how the events analysed were simulated, and depending on what you asked MA5 to plot precisely.

Best regards,

Benjamin
